README.md

```diff
@@ -879,7 +879,7 @@ In order to achieve greater coverage and encourage more people to contribute to
         <tr>
             <td><a href="https://en.wikipedia.org/wiki/Factorial">Factorial (Recursive)</a></td>
             <td> <!-- C -->
-                <a href="./src/c/FatorialRecursiva.c">
+                <a href="./src/c/RecursiveFactorial.c">
```

Comment:
Here too
```suggestion
                <a href="./src/c/FactorialRecursive.c">
```

Reply:
Here I really don't know what to do, if you use recursive as a prefix it will be close to recursive algorithms, but if you use factorial as a prefix, it will be next to the normal factorial and it would be interesting to have both side by side.

Reply:
Here too